### DISCUSSION TITLE
一些可能的新特性

### DISCUSSION THREAD

**Discussion by rayadaschn** [palxiao/poster-design#27] [Ideas]
- 项目的基础框架仍为 Vue3.0.0，许多较新的特新无法使用，是否能够升级到 Vue3.2+ ?
- 项目中加入了 less 和 sass 应当统一使用一种进行编译？

**Comment by palxiao** [comment]
项目中所有使用的css扩展都是less。

因为创建项目之初使用的element-plus还未发布正式版，其自动引用等功能绑定了sass、sass-node等库，后来经过手动升级element本项目也已剔除大部分依赖，devDependencies中的sass应该是漏网之鱼，当前已移除。

**Reply by rayadaschn** [reply]
此外，还是建议转向 Vue3.2 便于后续淘汰 Options API，和更新迭代。3.2 兼容性还是有的。

**Reply by palxiao** [reply]
本来我是打算从Vue2 -> Vue3，Vuex -> Pinia，这目前来说只是工作量的问题，毕竟一个人全量改起来太花时间，所以我都是做到哪部分再改到哪部分。

另外Vue当前最新稳定版是3.3，只更新到3.2的理由是什么呢？

**Reply by rayadaschn** [reply]
意思是 3.2 以上，是一个较大的版本迭代。3.3 也可以。

**Reply by palxiao** [reply]
后续考虑以一个新分支来做相关的升级测试
